docs/user_guide.md

```diff
@@ -163,10 +163,10 @@ Content-Length: 537
 }
 ```
 
-#### Incoming Webhook
+#### Job execution API
 
-We can execute job with incoming webhook. Incoming webhook uses basic authorization. We can set key-value on `api_basic_authentication_applications` in kuroko2.yml.
-To execute job from API, check `Allow API execution of this job` in job definition.
```

Comment:
Please remain this line.
`To execute job from API, check `Allow API execution of this job` in job definition.`